webui/src/components/Tree.js

```diff
@@ -327,7 +327,7 @@ const Tree = ({path, list, repo, refId, diffResults, onNavigate, onDelete, showA
     const refreshData = useCallback(() => {
         if (refId.type === 'branch') {
             if (refId.id === repo.default_branch) {
-                listBranches(repo.id, "", 2) // trigger list branches to
+                listBranches(repo.id, "") // trigger list branches to
```

Comment:
Will the amount param be sent at all, or is it empty("&amount=")?

Reply:
It will fallback to use this one - 
https://github.com/treeverse/lakeFS/blob/master/webui/src/actions/api.js#L4